var/spack/repos/builtin/packages/q-e-sirius/package.py

```diff
@@ -0,0 +1,315 @@
+
+    # Some QMCPACK converters are incompatible with upstream patches.
+    # HDF5 is a hard requirement. Need to do two HDF5 cases explicitly
+    # since Spack lacks support for expressing NOT operation.
+    conflicts(
+        '@6.4+patch',
+        when='+qmcpack',
+        msg='QE-to-QMCPACK wave function converter requires '
+        'deactivatation of upstream patches'
+    )
+    conflicts(
+        '@6.3:6.4.0 hdf5=serial',
+        when='+qmcpack',
+        msg='QE-to-QMCPACK wave function converter only '
+        'supported with parallel HDF5'
+    )
+    conflicts(
+        'hdf5=none',
+        when='+qmcpack',
+        msg='QE-to-QMCPACK wave function converter requires HDF5'
+    )
+
+    # The first version of Q-E to feature integrated EPW is 6.0.0,
+    # as per http://epw.org.uk/Main/DownloadAndInstall .
+    # Complain if trying to install a version older than this.
+    conflicts('+epw', when='@:5',
+              msg='EPW only available from version 6.0.0 and on')
+
+    # Below goes some constraints as shown in the link above.
+    # Constraints may be relaxed as successful reports
+    # of different compiler+mpi combinations arrive
+
+    # TODO: enable building EPW when ~mpi
+    conflicts('+epw', when='~mpi', msg='EPW needs MPI')
+
+    # EPW doesn't gets along well with OpenMPI 2.x.x
+    conflicts('+epw', when='^openmpi@2.0.0:2.999.999',
+              msg='OpenMPI version incompatible with EPW')
+
+    # EPW also doesn't gets along well with PGI 17.x + OpenMPI 1.10.7
+    conflicts('+epw', when='^openmpi@1.10.7%pgi@17.0:17.12',
+              msg='PGI+OpenMPI version combo incompatible with EPW')
+
+    # Spurious problems running in parallel the Makefile
+    # generated by the configure
+    parallel = False
+
+    def install(self, spec, prefix):
+
+        prefix_path = prefix.bin if '@:5.4.0' in spec else prefix
+        options = ['-prefix={0}'.format(prefix_path)]
+
+        sirius = spec['sirius']
+
+        options.append('LIBS={0}'.format(sirius.libs[0]))
+        options.append('LD_LIBS={0}'.format(sirius.libs[0]))
+
+        options.append('--disable-xml')
+
+        # QE autoconf compiler variables has some limitations:
+        # 1. There is no explicit MPICC variable so we must re-purpose
+        #    CC for the case of MPI.
+        # 2. F90 variable is set to be consistent with MPIF90 wrapper
+        # 3. If an absolute path for F90 is set, the build system breaks.
+        #
+        # Thus, due to 2. and 3. the F90 variable is not explictly set
+        # because it would be mostly pointless and could lead to erroneous
+        # behaviour.
+        if '+mpi' in spec:
+            mpi = spec['mpi']
+            options.append('--enable-parallel=yes')
+            options.append('MPIF90={0}'.format(mpi.mpifc))
+            options.append('CC={0}'.format(mpi.mpicc))
+        else:
+            options.append('--enable-parallel=no')
+            options.append('CC={0}'.format(env['SPACK_CC']))
```

Comment:
```suggestion
            options.append('CC={0}'.format(spack_cc))
```

Reply:
thanks, replaced also `env['SPACK_F77']` by `spack_f77` and the same for `env['SPACK_FC']`